keylime/config.py

```diff
@@ -234,11 +234,23 @@ def get_config(component: str) -> RawConfigParser:
     return _config[component]
 
 
+def _get_env(component: str, option: str, section: Optional[str]) -> Optional[str]:
+    env_name = f"KEYLIME_{component.upper()}"
+    if section:
+        env_name += f"_{section.upper()}"
+    env_name += f"_{option.upper()}"
+    return os.environ.get(env_name, None)
+
+
 def getlist(component: str, option: str, section: Optional[str] = None) -> List[Any]:
+    env_value = _get_env(component, option, section)
     if not section:
         section = component
 
-    read = get_config(component).get(section, option).strip('" ')
+    if env_value:
+        read = env_value.strip('" ')
```

Comment:
Since we're stripping out quotes and spaces in a couple places for env vars, should it be moved into `_get_env()`?

Reply:
We don't do it for the boolean, float and int conversions, so I don't think I can move into `_get_env()`.

Reply:
I guess I should have asked why we don't do it for those? I know spaces shouldn't really be there, but if people are going to accidentally quote strings, they might do it for ints/floats/booleans, especially if it's some automated config management thing happening.

Reply:
Good question. @ansasaki any inpunt on how this should be handled?

Reply:
I think it would be less error prone to remove quotes/spaces from all strings directly obtained from the config file.